code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -520,7 +520,7 @@ Contains:
 	flash_protect = 0
 	bang_protect = 0
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR
-	clothing_flags = STOPSPRESSUREDAMAGE | SHOWEROKAY | SNUG_FIT
+	clothing_flags = STOPSHIGHPRESSUREDMG | STOPSLOWPRESSUREDMG | SHOWEROKAY | SNUG_FIT
```

Comment:
Skinsuit shouldnt protect against high pressure

Reply:
Ah shit that was a replace all